.gitattributes

```diff
@@ -5,6 +5,8 @@
 *.bam           -text diff=bam
 
 # Omit these files from release tarballs.
+/.appveyor.yml  export-ignore
+/.cirrus.yml    export-ignore
 .git*           export-ignore
-.travis*        export-ignore
+.ci_helpers*    export-ignore
```

Comment:
At the risk of being very boring: the `*` here was so that it would match both _.travis_ and _.travis.yml_, so is no longer necessary; and `.travis` still appears in a comment in _.ci_helpers/clone_.

Reply:
Good spot.  I have made some adjustments.